### PR TITLE
PXC-4379 Release notes for Percona XtraDB Cluster 8.0.36-28

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/release-notes/8.0.36-28.md
	modified:   docs/release-notes/release-notes_index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]
Good
You could add the date

File: docs/release-notes/8.0.36-28.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+* [PXC-4341](https://perconadev.atlassian.net/browse/PXC-4341): When running `FLUSH TABLES` after a statement was prepared, the node could exit due to broken consistency.
+
+* [PXC-4348](https://perconadev.atlassian.net/browse/PXC-4348): The joiner node exited with Metadata Lock `BF-BF` conflict during `IST`.
+
+* [PXC-4362](https://perconadev.atlassian.net/browse/PXC-4362): The node could leave the cluster when binary logging was enabled and the function was created without super privilege.
+
+* [PXC-4365](https://perconadev.atlassian.net/browse/PXC-4365): The node could leave the cluster when the row size was too large and had more than three `nvarchar` columns.
+
+* [PXC-4380](https://perconadev.atlassian.net/browse/PXC-4380): If one of the nodes in a cluster with multiple nodes (for example, 15 nodes) had network issues, the cluster could become Non-Primary.
```
Comment (kamil-holubicki) [inline docs/release-notes/8.0.36-28.md]: This one was moved to 8.0.37

File: docs/release-notes/8.0.36-28.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+* The hashing algorithm employed yielded poor performance when using a HASH field to check for uniqueness. (Bug #109548, Bug #34959356)
+
+* All [statement instrument elements] that begin with `statement/sp/%`, except `statement/sp/stmt`, are disabled by default.
+
+Find the complete list of bug fixes and changes in the [MySQL 8.0.36 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-36.html).
+
+## Bug fixes
+
+* [PXC-4277](https://perconadev.atlassian.net/browse/PXC-4277): The Percona XtraDB Cluster cluster with three nodes became inconsistent when using `ALTER .. ALGORITHM=INPLACE`.
```
Comment (kamil-holubicki) [inline docs/release-notes/8.0.36-28.md]: Moved to 8.0.37

File: docs/release-notes/8.0.36-28.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+* [PXC-4348](https://perconadev.atlassian.net/browse/PXC-4348): The joiner node exited with Metadata Lock `BF-BF` conflict during `IST`.
+
+* [PXC-4362](https://perconadev.atlassian.net/browse/PXC-4362): The node could leave the cluster when binary logging was enabled and the function was created without super privilege.
+
+* [PXC-4365](https://perconadev.atlassian.net/browse/PXC-4365): The node could leave the cluster when the row size was too large and had more than three `nvarchar` columns.
+
+* [PXC-4380](https://perconadev.atlassian.net/browse/PXC-4380): If one of the nodes in a cluster with multiple nodes (for example, 15 nodes) had network issues, the cluster could become Non-Primary.
+
+* [PXC-4385](https://perconadev.atlassian.net/browse/PXC-4385): When running a `TOI ALTER USER` on one node and a `SET PASSWORD` on another, the statements hung and locked the affected nodes.
```
Comment (kamil-holubicki) [inline docs/release-notes/8.0.36-28.md]: Moved to 8.0.37

**Review by kamil-holubicki — CHANGES_REQUESTED** [review]
Additionally this ticket was included
https://perconadev.atlassian.net/browse/PXC-4340

File: docs/release-notes/8.0.36-28.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.36-28.md]: Removed

File: docs/release-notes/8.0.36-28.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.36-28.md]: Removed

File: docs/release-notes/8.0.36-28.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.36-28.md]: Removed

**Review by kamil-holubicki — APPROVED** [review]

**Pull request merged**